tests/test_rp_venv.py

```diff
@@ -63,6 +63,7 @@ def test_prepare_venv(rp_task_manager, sdist):
             'target': sdist_names[name],
             'action': rp.TRANSFER
         })
+    # TODO: Can we use `pilot://` for target file?
     logger.debug(str(input_staging))
     pilot.stage_in(input_staging)
```

Comment:
TODO: We can get rid of the staged RCT sdists once we confirm that the `rp_install` location is sufficient for the `prepare_env` scalems installation.

Reply:
Correction: prepare_env and Task environments do not automatically inherit access to the RCT stack and the raptor base classes. However, we can avoid staging the sdists *iff* venv_mode is not *local*. If we know that RP is going to *create* or *update* the Pilot venv, then we can count on the sdists being staged to the sandbox.

*Note: this is not actually the case in RP 1.14.*

Upshot: When preparing a Task venv, install the RCT stack.